src/main/java/nextstep/subway/common/CustomException.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.subway.common;
+
+public class CustomException extends RuntimeException {
```

Comment:
```suggestion
public abstract class CustomException extends RuntimeException {
```

상속을 위해 만드셨다면 추상 클래스로 만드는게 좋아보이네요 ㅎㅎ

Reply:
네 모든 커스텀익셉션의 상위 익셉션을 생각하고 만든거라서 추상 클래스로 선언하면 좋을것 같습니다. 감사합니다.